devfile.yaml

```diff
@@ -0,0 +1,25 @@
+schemaVersion: 2.1.0
+metadata:
+  name: catalog-spring-boot
+attributes:
+  che-theia.eclipse.org/sidecar-policy: mergeImage
+  metadata-name-field: generateName
+  metadata-name-original-value: catalog-spring-boot
+projects:
+  - git:
+      remotes:
+        origin: 'https://github.com/evanshortiss/catalog-spring-boot.git'
```

Comment:
Need to change this back to the correct origin before/after merging if we think this is an OK solution.